src/renderer/features/governance/ReferendumFilter/lib/referendum-filter-utils.ts

```diff
@@ -0,0 +1,7 @@
+export const referendumFilterUtils = {
+  test,
+};
+
+function test() {
+  console.log(1);
```

Comment:
do you need it as part of the POC?

Reply:
It's a boilerplate for the future code
It might be removed completely when the actual task will be developed